src/main/java/at/ac/tuwien/sese/g09/service/mapper/RoomMapper.java

```diff
@@ -0,0 +1,35 @@
+package at.ac.tuwien.sese.g09.service.mapper;
+
+import at.ac.tuwien.sese.g09.domain.Room;
+import at.ac.tuwien.sese.g09.domain.RoomCapacity;
+import at.ac.tuwien.sese.g09.domain.RoomPicture;
+import at.ac.tuwien.sese.g09.domain.RoomPrice;
+import at.ac.tuwien.sese.g09.service.dto.RoomResponseDTO;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
I think this should not be a service, is it possible to use `@Mapper`?

Reply:
Otherwise i would suggest to not use any annotation and simply create an instance of the mapper where needed.

Reply:
I used the same annotation already in use in UserMapper

Reply:
Right my bad should also be changed there i guess. However if it is to complicated, no change needed.

Reply:
Why do you want to remove the DI for the Mapper?

A Mapper provides me the service of mapping an object to another object. Therefore, In my opinion, the annotation @Service is correct